bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -112,14 +112,14 @@ metadata:
     capabilities: Full Lifecycle
     categories: AI/Machine Learning, Big Data
     certified: "False"
-    containerImage: quay.io/opendatahub/opendatahub-operator:v2.7.0
+    containerImage: quay.io/opendatahub/opendatahub-operator:v2.8.0
     createdAt: "2024-2-06T00:00:00Z"
-    olm.skipRange: '>=1.0.0 <2.0.0'
+    olm.skipRange: '>=1.0.0 <2.8.0'
```

Comment:
we want to have a one jump from 1.33 to 2.8 directly?

Reply:
I will revert this change